numba/core/typing/typeof.py

```diff
@@ -11,6 +11,11 @@
 # terminal color markup
 _termcolor = errors.termcolor()
 
+try:
```

Comment:
This can probably do with a comment, why two code paths are needed.

Reply:
There was a discrepancy in module names from version to version. (They renamed the module in a newer version, hence I had to account for both of them)